thirdeye/thirdeye-pinot/src/test/java/org/apache/pinot/thirdeye/util/ThirdEyeUtilsTest.java

```diff
@@ -176,21 +179,51 @@ public void testExceptionToStringLimited() {
   }
 
   @Test
-  public void testMergeAnomalyProperties() {
+  public void testMergeAnomalyProperties() throws Exception {
+    final long now = System.currentTimeMillis();
+    final long bucketSize = 300_000;
+    final double parentVal = 1.0, childVal = 2.0;
+
     Map<String, String> parentProperties = new HashMap<>();
     parentProperties.put("p1", "value1");
     parentProperties.put("p2", "value2");
     parentProperties.put("detectorComponentName", "rule1");
+    parentProperties.put("anomalyTimelinesView",
+        generateAnomalyTimelineView(now, bucketSize, 10, parentVal).toJsonString());
+
 
     Map<String, String> childProperties = new HashMap<>();
     childProperties.put("p1", "value3");
     childProperties.put("c1", "value4");
     childProperties.put("detectorComponentName", "rule2");
+    childProperties.put("anomalyTimelinesView",
+        generateAnomalyTimelineView(now + bucketSize, bucketSize, 10,  childVal).toJsonString());
```

Comment:
Can we add another unit test case where the child anomaly has a gap with parent? say 

`childProperties.put("anomalyTimelinesView", generateAnomalyTimelineView(now + bucketSize * 20, bucketSize, 10,  childVal).toJsonString());`

Reply:
Added it as a new test.